Cargo.toml

```diff
@@ -20,12 +20,12 @@ spatial_index = ["dep:rstar"]
 
 [dependencies]
 ahash = "0.8"
-arrow2 = { version = "0.17", features = [] }
-geo = "0.26"
-geoarrow = { package = "geoarrow2", version = "0.0.1", optional = true, features = ["geozero"] }
+arrow = "49.0.0"
+geo = "0.27"
+geoarrow = { package = "geoarrow2", version = "0.0.2", optional = true, features = ["geozero"] }
```

Comment:
You can use `geoarrow = "0.1"` now

Reply:
Great, I did not notice the crate moved to "geoarrow" instead of "geoarrow2". Nice.